composer.json

```diff
@@ -11,15 +11,14 @@
         "drupal/core": "^10",
         "drupal/datetime_range_timezone": "^1.0.0-alpha3",
         "drupal/linkit": "^6.0.0-beta4",
-        "drupal/maxlength": "^2.0.1",
+        "drupal/maxlength": "dev-3.x",
```

Comment:
I think it might be better to point to a specific commit instead of having dev

Reply:
It is temporary until we get the new release, in order to have dependencies handled correctly.